applications/openshift/helper/version_detect/rule.yml

```diff
@@ -0,0 +1,22 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'This is a helper rule to fetch the required api resource for detecting OCP version'
+
+{{% set custom_jqfilter = '{{.ocp_version_yaml_path}}' %}}
+{{% set custom_api_path = '{{.ocp_version_api_path}}' %}}
+{{% set default_api_path = '/apis/config.openshift.io/v1/clusteroperators/openshift-apiserver' %}}
+{{% set dump_path = default_api_path ~ ',' ~ custom_jqfilter %}}
+
+description: |-
+    no description
+
+rationale: |-
+    no rationale
+
+severity: info
+
+warnings:
+- general: |-
+    {{{ openshift_filtered_version({custom_api_path: dump_path}) | indent(4) }}}
```

Comment:
Making sure that I understand this part and the jinja part correctly: Does this say that we will save `custom_api_path` filtered through `custom_jqfilter` but regardless of their actual value, we will pretend that they come from `/apis/config.openshift.io/v1/clusteroperators/openshift-apiserver` ?

Reply:
By pretend, I mean save in the data dump at location normally used for `/apis/config.openshift.io/v1/clusteroperators/openshift-apiserver` ?

Reply:
that's correct! And we just need to adjust those two variables in a HyperShift Env

Reply:
that's correct! And we just need to adjust those two variables in a HyperShift Env

Reply:
just changed the data dump location to `/ocp/version` to be backward consistent